### DISCUSSION TITLE
Android 15

### DISCUSSION THREAD

**Discussion by vl11111** [AndreyPavlenko/Fermata#550] [General]
Google Pixel (all series) are rolling out Android 15. I have Pixel 8 PRO and 9 PRO already updated.
As I understand since we are still screwing A14, A15 will get Fermata Auto like never?)))

**Comment by AndreyPavlenko** [comment]
I've not tested, but I don't expect any difference from A14.

**Reply by vl11111** [reply]
А какой у тебя вообще личнo телефон?

**Comment by vl11111** [comment]
https://xdaforums.com/t/add-fermata-screen-mirroring-to-android-auto-in-android-14-root-non-root.4692350/

**Reply by vl11111** [reply]
did anyone try?

**Reply by JVKeller** [reply]
This is nothing new and has been discussed lots of times in this repo... 

I use the non root method to get screen mirroring to work with current versions. It's finicky, but it will work!

**Reply by vl11111** [reply]
can you pinpoint which one? I tried the method above, but when i press self mode in headunit reloaded it just freezes or closes

**Reply by JVKeller** [reply]
I did pinpoint :) The non root method. Will HUR load for you if you turn off BT and WIFI? It's finiky and crashes and freezes often. 
You can try turning off/on hardware acceleration in the graphics setting, and change the pixel density or it won't fit on the phone screen very well. I have mine at 240

**Reply by vl11111** [reply]
You mean non root method from the link above?
